test/model/ShoppingCart.spec.js

```diff
@@ -4,66 +4,62 @@ import ShoppingCart from "../../src/model/ShoppingCart.js";
 
 describe("Shopping cart should checkout", () => {
 
-    it("Should calculate correct total and loyalty points for 10% discounted products", () => {
-        const customer = new Customer("Test customer");
+    let customer;
+    beforeEach(() => {
+        customer = new Customer("Test customer")
+    })
```

Comment:
To keep with same code style in terms of syntax, I think we are missing a semicolon here.

```suggestion
    });
```